src/validator.cpp

```diff
@@ -952,63 +962,131 @@ void Validator::ValidatorImpl::validateMathMLElements(const XmlNodePtr &node, co
     }
 }
 
-void Validator::ValidatorImpl::validateConnections(const ModelPtr &model)
+/**
+ * @brief Test to see if the given variables are reachable from their parent components.
+ *
+ * Determine if the variables parents are reachable from one another.  That is they either
+ * have a parent/child relationship or a sibling relationship in the models component
+ * hierarchy.
+ *
+ * Both variables must have a valid parent.
```

Comment:
Is it enough to just say it here? you don't need to test it?

Reply:
That is why it is said because it is not being tested.